pyproject.toml

```diff
@@ -52,6 +52,8 @@ pytest-sugar = "^0.9.2"
 click = "^8.0"
 freezegun = "^1.2.2"
 pytest-recording = "^0.12.1"
+# pin for cos-python-sdk-v5 issue
+requests = "2.27.1"
```

Comment:
cos-python-sdk-v5 has a max requests version of 2.27.1: https://github.com/tencentyun/cos-python-sdk-v5/blob/master/setup.py#L19-L21

while base c7n doesnt require requests for its main deps other packages do so this prevents flapping